boards/ambiq/apollo3p_evb/apollo3p_evb.dts

```diff
@@ -0,0 +1,71 @@
+/dts-v1/;
+#include <ambiq/ambiq_apollo3p_blue.dtsi>
+
+#include "apollo3p_evb-pinctrl.dtsi"
+
+/ {
+	model = "Ambiq Apollo3 Blue Plus evaluation board";
+	compatible = "ambiq,apollo3p_evb";
+
+	chosen {
+		zephyr,itcm = &tcm;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,uart-pipe = &uart0;
+	};
+
+	aliases {
+		watchdog0 = &wdt0;
+	};
+};
+
+&uart0 {
+	current-speed = <115200>;
+	pinctrl-0 = <&uart0_default>;
+	pinctrl-names = "default";
+	status = "okay";
+};
+
+&counter0 {
+	status = "disabled";
+};
+
+&wdt0 {
+	status = "disabled";
+};
```

Comment:
Makes no sense, peripherals should be disabled at the .dtsi level in the dts folder, if it needs disabling here then this is wrong, they should already be disabled and be here if they need to be enabled @gmarull